README.md

```diff
@@ -1,26 +1,66 @@
-# Shared resources for Hypothesis front-end applications
+# `@hypothesis/frontend-shared`
 
-A package of resources for Hypothesis front-end applications.
+UI components for Hypothesis front-end applications.
 
-## Requirements
+_Note_: This package currently provides two sets of components:
 
-Your project must have Preact installed as a dependency.
+- **Legacy components**: a complete set is available, imported from `@hypothesis/frontend-shared/`. These components are styled with SASS and have some external assets.
+- **Components**: An incomplete "updated" set is available, imported from `@hypothesis/frontend-shared/lib/next`. These components are styled with `tailwindcss` and have no external assets.
 
-`npm install --save preact`
+Legacy components are deprecated and are slated for removal at this package's next major version (`v6.0`).
 
-## Usage
+## Installation
+
+Your project must have `preact` and `tailwindcss` as dependencies.
+
+```sh
+$ yarn add preact tailwindcss
+$ yarn add @hypothesis/frontend-shared
+```
 
+_Note_: To use legacy components, your project must also have `sass` as a dependency.
+
+### tailwindcss configuration
+
+Update your project's tailwind configuration:
+
+```js
+import tailwindConfig from '@hypothesis/frontend-shared/lib/tailwind.preset.js';
+
+export default {
+  // Use this package's preset
+  presets: [tailwindConfig],
+  content: [
+    // Be sure to add this project's component source to your
+    // tailwind content globs
+    './node_modules/@hypothesis/frontend-shared/lib/**/*.js',
+  ],
+  // ...
 ```
-$ npm install @hypothesis/frontend-shared --save
+
+## Documentation
+
+Full documentation is available in this project's web-based pattern library.
+
+1. Run the local web server:
+   ```sh
+   $ make dev
+   ```
+2. Visit http://localhost:4001/ in a browser
+
+## Usage
+
+```js
+import { Link } from '@hypothesis/frontend-shared/lib/next';
 ```
 
-### Components (preact)
+## Usage (legacy)
 
 ```js
-import { LabeledButton } from '@hypothesis/frontend-shared';
+import { Link } from '@hypothesis/frontend-shared';
 ```
 
-### Icons
+### Icons (legacy)
```

Comment:
I noticed the syntax for legacy icon usage is invalid:

```js
export default function MyComponent() => {
```

This mixes arrow and non-arrow function syntax.

Reply:
Oh, thanks!